CMakePresets.json

```
@@ -40,6 +40,24 @@
         "BUILD_CMOCKA_LIB": true
       }
     },
+    {
+      "name": "cheribsd",
+      "inherits": "default",
+      "description": "CheriBSD build (alpha)",
+      "toolchainFile": "${sourceDir}/CMakeModules/CMakeToolchains/cheri.cmake",
+      "cacheVariables": {
+        "BUILD_MNL_LIB": false,
+        "BUILD_NETLINK_LIB": false,
+        "USE_NETLINK_SERVICE": false,
+        "USE_UCI_SERVICE": false,
+        "USE_GENERIC_IP_SERVICE": true,
+        "USE_RADIUS_SERVICE": true,
+        "BUILD_HOSTAPD": false,
+        "USE_CRYPTO_SERVICE": false,
+        "BUILD_OPENSSL_LIB": false,
+        "BUILD_CMOCKA_LIB": true
+      }
+    },
```

You can use `"inherits": "freebsd",` to copy most of the config from there.

```suggestion
    {
      "name": "cheribsd",
      "inherits": "freebsd",
      "description": "CheriBSD build (alpha)",
      "toolchainFile": "${sourceDir}/CMakeModules/CMakeToolchains/cheri.cmake"
    },
```